change/@ni-nimble-blazor-a91b538b-3c3f-49a8-bc97-2a4f98086198.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Once the 13 upgrade goes in I'm guessing the diffs for all these packages go away. If so, reminder to delete the change files for any packages not changing.

If there are still changes to these packages then as I did in the 13 PR, I'd encourage changing the comment to be more specific for the changes to non-Angular packages.

Reply:
Good point, will do.